test/metabase/api/downloads_exports_test.clj

```diff
@@ -1061,32 +1061,72 @@
                                                        :type     :native
                                                        :native   {:query "SELECT 1234.567 as A"}}}]
         (testing "CSV downloads respect the formatted/unformatted setting"
-          (let [formatted-json-results   (all-downloads card {:export-format :csv :format-rows true})
-                unformatted-json-results (all-downloads card {:export-format :csv :format-rows false})]
+          (let [formatted-results   (all-downloads card {:export-format :csv :format-rows true})
+                unformatted-results (all-downloads card {:export-format :csv :format-rows false})]
             (is (= {:unsaved-card-download    [["A"] ["1,234.57"]]
                     :card-download            [["A"] ["1,234.57"]]
                     :public-question-download [["A"] ["1,234.57"]]
                     :dashcard-download        [["A"] ["1,234.57"]]
                     :public-dashcard-download [["A"] ["1,234.57"]]}
-                   formatted-json-results))
+                   formatted-results))
             (is (= {:unsaved-card-download    [["A"] ["1234.567"]]
                     :card-download            [["A"] ["1234.567"]]
                     :public-question-download [["A"] ["1234.567"]]
                     :dashcard-download        [["A"] ["1234.567"]]
                     :public-dashcard-download [["A"] ["1234.567"]]}
-                   unformatted-json-results))))
+                   unformatted-results))))
         (testing "JSON downloads respect the formatted/unformatted setting"
-          (let [formatted-json-results   (all-downloads card {:export-format :json :format-rows true})
-                unformatted-json-results (all-downloads card {:export-format :json :format-rows false})]
+          (let [formatted-results   (all-downloads card {:export-format :json :format-rows true})
+                unformatted-results (all-downloads card {:export-format :json :format-rows false})]
             (is (= {:unsaved-card-download    [["A"] ["1,234.57"]]
                     :card-download            [["A"] ["1,234.57"]]
                     :public-question-download [["A"] ["1,234.57"]]
                     :dashcard-download        [["A"] ["1,234.57"]]
                     :public-dashcard-download [["A"] ["1,234.57"]]}
-                   formatted-json-results))
+                   formatted-results))
             (is (= {:unsaved-card-download    [["A"] [1234.567]]
                     :card-download            [["A"] [1234.567]]
                     :public-question-download [["A"] [1234.567]]
                     :dashcard-download        [["A"] [1234.567]]
                     :public-dashcard-download [["A"] [1234.567]]}
-                   unformatted-json-results))))))))
+                   unformatted-results))))))))
+
+(deftest pivot-measures-order-test
```

Comment:
is it worth adding tests for rows/columns/measures independently? Might be nice if one of them fails to figure out what's going on sooner.